tutorials/ndmg_bids.md

```diff
@@ -16,19 +16,19 @@ if not op.isdir(data_dir):
 
 # Run ndmg with docker from the command-line
 ```{bash}
-input_dir = $HOME/.ndmg/HNU1t
+export input_dir=$HOME/.ndmg/HNU1t
 mkdir $HOME/.ndmg/output
 
 # Option A (Docker executable approach):
-docker run -ti --rm --privileged -e DISPLAY=$DISPLAY -v $HOME/.ndmg/HNU1t:/input -v $HOME/.ndmg/output:/outputs ndmg_dev:latest --participant_label 0025427 --session_label 1 --modality dwi --atlas desikan --mod det --tt local --mf csd --sp native --seeds 20 /input /output participant
+docker run -ti --rm --privileged -e DISPLAY=$DISPLAY -v $HOME/.ndmg/HNU1t:/input -v $HOME/.ndmg/output:/outputs ndmg_dev:latest --participant_label 0025427 --session_label 1 --atlas desikan --mod det --tt local --mf csd --sp native --seeds 20 /input /output
```

Comment:
I think for these tutorials we should have the first command be as simple as possible: `ndmg input output`, and then we can start showing what the various flags do as people progress through the tutorials.

But I can add that today when I'm going through them